tests/third_party/cupy/fft_tests/test_fft.py

```diff
@@ -142,8 +142,8 @@ def test_ifft(self, xp, dtype):
                 # {"shape": (0, 5), "s": None, "axes": None}, # mkl_fft gh-110
                 # {"shape": (2, 0, 5), "s": None, "axes": None}, # mkl_fft gh-110
                 # {"shape": (0, 0, 5), "s": None, "axes": None}, # mkl_fft gh-110
-                {"shape": (3, 4), "s": (0, 5), "axes": None},
-                {"shape": (3, 4), "s": (1, 0), "axes": None},
+                {"shape": (3, 4), "s": (0, 5), "axes": (0, 1)},
```

Comment:
can we add a comment, like:
```suggestion
                # if s is given, axes can not be None
                {"shape": (3, 4), "s": (0, 5), "axes": (0, 1)},
```

Reply:
I have already added a comment that some of the tests are updated. 
https://github.com/IntelPython/dpnp/blob/6cd9389804839edb825152e30248eab9b9921dd6/tests/third_party/cupy/fft_tests/test_fft.py#L123-L125

I do not think we need to indicate it individually for each test. Besides, I believe sooner or later `cupy` will update the tests.

Reply:
I see, I missed that. Then I agree there is no reason to duplicate with extra comment few lines later.